getusermedia.html

```diff
@@ -2332,230 +2332,68 @@ <h2>MediaStreams in Media Elements</h2>
+                argument.</p>
+              </li>
+              <li>
+                <p>Let <var>message</var> be the constructor's second argument.
+                </p>
+              </li>
+              <li>
+                <p>Let <var>e</var> be a new
+                <code><a>OverconstrainedError</a></code> object.</p>
+              </li>
+              <li>
+                <p>Invoke the <code><a>DOMException</a></code> constructor of
+                <var>e</var> with the <code>message</code> argument set to
+                <var>message</var> and the <code>name</code> argument set to
+                <code>"OverconstrainedError"</code>.</p>
+                <p class="note">This name does not have a mapping to a legacy
+                code so <var>e</var>'s <code>code</code> attribute will return
+                0.</p>
+              </li>
+              <li>
+                <p>Set <var>e.constraint</var> to <var>constraint</var>.</p>
+              </li>
+              <li>
+                <p>Return <var>e</var>.</p>
+              </li>
+            </ol>
+          </dd>
+        </dl>
       </section>
       <section>
-        <h2>Properties of the OverconstrainedError Prototype Object</h2>
-        <p>The OverconstrainedError prototype object is an ordinary object. It
-        is not an Error instance and does not have an [[\ErrorData]] internal
-        slot.</p>
-        <p>The value of the [[\Prototype]] internal slot of the
-        OverconstrainedError prototype object is the intrinsic object
-        %ErrorPrototype%.</p>
-        <section>
-          <h2>OverconstrainedError.prototype.constructor</h2>
-          <p>The initial value of the constructor property of the prototype for
-          the OverconstrainedError constructor is the intrinsic object <a href=
-          "#error-handling">%OverconstrainedError%</a>.</p>
-        </section>
-        <section>
-          <h2>OverconstrainedError.prototype.constraint</h2>
-          <p>The initial value of the constraint property of the prototype for
-          the OverconstrainedError constructor is the empty String.</p>
-        </section>
-        <section>
-          <h2>OverconstrainedError.prototype.message</h2>
-          <p>The initial value of the message property of the prototype for the
-          OverconstrainedError constructor is the empty String.</p>
-        </section>
-        <section>
-          <h2>OverconstrainedError.prototype.name</h2>
-          <p>The initial value of the name property of the prototype for the
-          OverconstrainedError constructor is
-          <code>"<b>OverconstrainedError</b>"</code>.</p>
-        </section>
+        <h2>Attributes</h2>
+        <dl data-link-for="OverconstrainedError" data-dfn-for="OverconstrainedError" class="attributes">
+          <dt><dfn data-idl><code>constraint</code></dfn> of type <span
+          class="idlAttrType"><a>DOMString</a></span>, readonly</dt>
```

Comment:
```suggestion
          class="idlAttrType">{{DOMString}}</span>, readonly</dt>
```
(this will fix the CI error)